test/spec/events_spec.js

```diff
@@ -219,6 +219,11 @@ define(['lib/component', 'lib/registry', 'lib/dom'], function (defineComponent,
       instance2.on(document, 'click', spy);
       instance1.trigger('click');
       expect(spy).toHaveBeenCalled();
+
+      spy = jasmine.createSpy();
```

Comment:
Could you set this up as a separate test, perhaps that has a list of elements and generates a test for  each of them? `[window, document, 'head', 'body'].forEach(...)`